spec.md

```diff
@@ -768,7 +769,10 @@ For success responses, the following fields are defined:
 | Response Field | Type | Description |
 | --- | --- | --- |
 | state* | string | Valid values are `in progress`, `succeeded`, and `failed`. While `"state": "in progress"`, the Platform SHOULD continue polling. A response with `"state": "succeeded"` or `"state": "failed"` MUST cause the Platform to cease polling. |
+
 | description | string | A user-facing message that can be used to tell the user details about the status of the operation. |
+| status_code | number | The HTTP status code that would have been returned if the operation would have been executed synchronously. If the state is `failed` this field SHOULD be present and the value MUST be an integer in the range of 400 to 599. This field MUST NOT be present for any other state. |
+| error | string | An error code as described in the [Service Broker Errors](#service-broker-errors) section. If present, MUST be a non-empty string. If the state is `failed` and there is an error code this field SHOULD be present. This field MUST NOT be present for any other state. |
```

Comment:
Which error code are we actually expecting to get back here?
`AsyncRequired` = not applicable
`ConcurrencyError` = should be returned to the original request with a `422` as outlined in the Blocking Operations section (surely the broker can decide at that time whether or not the instance is already being operated on?)
`RequiresApp` = not applicable

Reply:
You have listed the codes the spec defines and none of them might be applicable here. But a broker can make up its own error code if the spec doesn’t define one.
But the main reason for this field is for parity with an error messages of a synchronous call.